server/runtime.go

```diff
@@ -177,6 +179,24 @@ func (s *Server) CreatePodSandbox(ctx context.Context, req *pb.CreatePodSandboxR
 
 	s.addContainer(container)
 
+	meta := &metadata{
+		LogDir:        logDir,
+		ContainerName: containerName,
+		Labels:        labels,
+	}
+
+	b, err := json.Marshal(meta)
+	if err != nil {
+		return nil, err
+	}
+
+	// TODO: eventually we would track all containers in this pod so on server start
```

Comment:
alright, yeah, everything comes from `config.json` so there won't be the need for the metadata file in containers (probably gonna remove it in sandbox if it isn't needed there as well)